### PR TITLE
Index all items + expanded attributes

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apollos-api/src/data/content-items/data-source.js [hunk 1]
```diff
@@ -70,6 +70,17 @@ class ExtendedContentItem extends ContentItem.dataSource {
     return theme;
   }
 
+  bySearchableContent = () =>
```
Comment (vinnyjth) [inline packages/apollos-api/src/data/content-items/data-source.js]: We should port something like this to core.

File: packages/apollos-api/src/data/search/data-source.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      const items = result.map(({ node }) => node);
+      itemsLeft = items.length === 100;
+
+      if (itemsLeft) args.after = result[result.length - 1].cursor;
+
+      const indexableItems = await Promise.all(
+        items.map((item) => this.mapItemToAlgolia(item))
+      );
+
+      await this.addObjects(indexableItems.filter((item) => !!item));
```
Comment (vinnyjth) [inline packages/apollos-api/src/data/search/data-source.js]: Need to port the addition of this `.filter` to core.

**Comment by conrad-vanl** [comment]
Is this ready?

**Comment by vinnyjth** [comment]
@conrad-vanl Yep, this is ready for review.

**Comment by vinnyjth** [comment]
Merging.
